deploy/README.md

```diff
@@ -65,41 +65,44 @@ This approach requires access to the Sumo Logic Collector API. It will create a
 
 ```sh
 curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/setup.sh \
-  | bash -s <api-endpoint> <access-id> <access-key> [collector-name]
+  | bash -s [-c collector-name] [-k cluster-name] <api-endpoint> <access-id> <access-key>
 ```
 
 __NOTE__ This script will be executed in bash and requires [jq command-line JSON parser](https://stedolan.github.io/jq/download/) to be installed.
 
 #### Parameters
 
+* __-c collector-name__ - optional. Name of Sumo collector that will be created. If not specified, it will be named as `kubernetes-<timestamp>`
+* __-k cluster-name__ - optional. Name of the Kubernetes cluster that will be attached to logs as metadata. If not specified, it will be named as `kubernetes-<timestamp>`
```

Comment:
so this change only apply to log? (how about metrics/event?)

Reply:
`cluster name` attaches the `fields:{cluster:CLUSTER_NAME}` to the metadata of the collector, which gets attached to all the logs sent to that collector. Actually since we receive events as logs in Sumo, this should also apply for events. For metrics, you have to set the same `CLUSTER_NAME` in the prometheus operator `overrides.yaml`
```
prometheus:
  prometheusSpec:
    externalLabels:
      # Set this to a value to distinguish between different k8s clusters
      cluster: kubernetes
```

Eventually this value can be enforced the same in the helm chart when we release that.